src/main/java/io/pinecone/unsigned_indices_model/QueryResponseWithUnsignedIndices.java

```diff
@@ -52,4 +57,30 @@ public String getNamespace() {
     public void setNamespace(String namespace) {
         this.namespace = namespace;
     }
+
+    public Usage getUsage() { return usage; }
+
+    public void setUsage(Usage usage) { this.usage = usage; }
+
+    /**
+     * Convert the given object to string with each line indented by 4 spaces
+     * (except the first line).
+     */
+    private String toIndentedString(Object o) {
```

Comment:
I lifted these functions from some of our generated code just to make things a bit more readable.